storage/miner.go

```diff
@@ -109,7 +109,11 @@ func (m *Miner) Run(ctx context.Context) error {
 	}
 
 	go fps.run(ctx)
-	go m.sectorStateLoop(ctx)
+	if err := m.sectorStateLoop(ctx); err != nil {
```

Comment:
This must be in a goroutine, otherwise you can only process one sector at a time (including sealing)

Reply:
The function does some initial checks and then spins off goroutines internally. Are you sure?

Reply:
Ohwait, I thought it's a different `go something()` call